calico/felix/config.py

```diff
@@ -325,9 +331,20 @@ def _finish_update(self, final=False):
         self.MAX_IPSET_SIZE = self.parameters["MaxIpsetSize"].value
         self.IPTABLES_GENERATOR_PLUGIN = \
             self.parameters["IptablesGeneratorPlugin"].value
+	self.IPTABLES_MARK_MASK =\
+	    self.parameters["IptablesMarkMask"].value
 
         self._validate_cfg(final=final)
 
+        # Now the config has been validated, generate the IPTables mark masks
+        # we'll actually use internally.
+        mark_mask = self.IPTABLES_MARK_MASK
+
+        # Extract the least significant bit and use it as the accept mask.
+        next_mask = mark_mask & (mark_mask - 1)
+        self.IPTABLES_MARK_ACCEPT = mark_mask - next_mask
```

Comment:
I'd be tempted to convert this to hex so that it's easier to understand when we log out what we're about to program into iptables.  WDYT?